core/assets/bundles/bundle.properties

```
@@ -832,6 +832,7 @@ setting.chatopacity.name = Chat Opacity
 setting.lasersopacity.name = Power Laser Opacity
 setting.bridgeopacity.name = Bridge Opacity
 setting.playerchat.name = Display Player Bubble Chat
+setting.enableweather.name = Enable Weather
```

```suggestion
setting.enableweather.name = Show Weather Graphics
```
You aren't actually toggling the weather system itself, just the graphics. This should make it clear.